sender:
  ros__parameters:
    pub_frequency: 1
    ping_pub_topic: "ping_msg"
    ping_sub_topic: "pong_msg"
    
    msg_num:  30

    qos_depth : 1000
    qos_best_effort: True
    qos_durability_transient_local: False
    qos_history_keep_all: False

receiver:
  ros__parameters:
    pong_pub_topic: "pong_msg"
    pong_sub_topic: "ping_msg"

    qos_depth:  1000
    qos_best_effort: True
    qos_durability_transient_local: False
    qos_history_keep_all: False

